agent-core/src/test/java/com/newrelic/agent/android/logging/AgentLogManagerTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023. New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.newrelic.agent.android.logging;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class AgentLogManagerTest {
```

Comment:
Oh, I guess this was `TODO`. It will come up during your work @ywang-nr, so maybe you could fill it in?

Reply:
Yes, I can cover this part.